lib/components/feed/postFullDetails.tsx

```
@@ -33,61 +39,78 @@ function PostFullDetails({ post, onClose }: postFullDetailsProps) {
 	return (
-		<ThemedView type="background" style={styles.proposerInfo}>
+		<ThemedView type="background" style={[styles.proposerInfo, styles.flexRowJustifySpaceBetween]}>
 			<ThemedText type="bodyLarge">Benificiaries</ThemedText>
-			{benificiaries.map((benificary) => (
-				<View style={{ flexDirection: "row", gap: 8, alignItems: "center" }}>
-					<OnChainUserInfo key={benificary.address} address={benificary.address} />
-				</View>
-			))}
+			<View style={{ flexDirection: "column", gap: 8 }}>
+				{benificiaries.map((benificary) => (
+					<View style={{ flexDirection: "row", gap: 8, alignItems: "center" }}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)